build-python.sh

```diff
@@ -17,11 +17,8 @@
 set -e
 
 : ${LIBFFI_VERSION:=v3.4.2}
-: ${PYTHON_MAJOR:=3}
-: ${PYTHON_MINOR:=9}
-: ${PYTHON_PATCH:=7}
-: ${PYTHON_VERSION:=v${PYTHON_MAJOR}.${PYTHON_MINOR}.${PYTHON_PATCH}}
```

Comment:
This removes `$PYTHON_VERSION` which is needed for checking out the matching upstream vanilla python version, used for the native unix build.

Reply:
Sorry, I misread, as that variable still is present and you just got rid of other less needed ones. Then it's indeed fine.

Testing this new form now, then merging afterwards. If you want to, you can squash the two commits together, otherwise I'll use github's "squash and merge" (or maybe squash locally and push that way).